swan-lake/integration-tools/edi-tool.md

```diff
@@ -19,22 +19,22 @@ The functionalities of the above usages are described below.
 
 ### Code generation usage
 
-The below command generates all Ballerina records and parsing functions required for working with data in the given EDI schema and writes those into the file specified in the output path.
+The command below generates all Ballerina records and parsing functions required for working with data in the given EDI schema and writes those into the file specified in the output path.
 
 ```
 $ bal edi codegen <edi-schema-path> <output-path>
 ```
 
-The generated parsing function (i.e., `fromEdiString(...)`) can read EDI text files into generated records, which can be accessed from the Ballerina code, similar to accessing any other Ballerina record. Similarly, the generated serialization function (i.e., `toEdiString(...)`) can serialize generated Ballerina records into EDI text.
+The generated parsing function (i.e., `fromEdiString(...)`) can read EDI text files into generated records, which can be accessed from the Ballerina code similar to accessing any other Ballerina record. Similarly, the generated serialization function (i.e., `toEdiString(...)`) can serialize generated Ballerina records into EDI text.
 
 ### Package generation usage
 
-Usually, organizations have to work with many EDI formats, and integration developers need to have a convenient way to work on EDI data with minimum effort. The Ballerina EDI package facilitates this by allowing organizations to pack all EDI processing codes of their EDI collections into an importable package. Therefore, integration developers can simply import the package and convert EDI messages into Ballerina records in a single line of code.
+Usually, organizations have to work with many EDI formats and integration developers need to have a convenient way to work on EDI data with minimum effort. The Ballerina EDI package facilitates this by allowing organizations to pack all EDI processing codes of their EDI collections into an importable package. Therefore, integration developers can simply import the package and convert EDI messages into Ballerina records in a single line of code.
 
-The below command can be used to generate the EDI package.
+The command below can be used to generate the EDI package.
 
 ```
-$ bal edi libgen -O <org-name> -n <package-name> -s <edi-schema-folder> -o <output-folder>
+$ bal edi libgen -O <org-name> -n <package-name> -s <edi-schema-path> -o <output-path>
 ```
 
 A Ballerina package project will be generated in the output folder. This package can be built and published by issuing the `bal pack` and `bal push` commands from the output folder. Then, the generated package can be imported into any Ballerina project, and the generated utility functions of the package can be invoked to parse EDI messages into Ballerina records.
```

Comment:
`A ballerina package project` is not meaninful.

Reply:
Changed to `package`.